pegasus/sites.v3/hourofcode.com/views/home_highlights.haml

```diff
@@ -43,6 +43,19 @@
         .clear
         %a{href: "/highlights"}
           %button See more highlights
+-elsif latam_country_code?(@country) || @country == 'la'
+  -# For LATAM countries, show only the HoC Spanish Twitter feed.
+  -# For historical reasons, we also use 'la' code to represent LATAM countries in
+  -# i18n/countries.json. Unfortunately, it is a real country code for Laos Republic
+  -# (ISO_3166-1_alpha-2 codes). We should change it to a different code later.
+  .row
+    .col-md-8
+      %br/
+      %h2
+        Hour of Code Highlights
+      %a.twitter-grid{data: {tweet: {limit: "1"}, chrome: "noheader nofooter"}, href: "https://twitter.com/codeorg/timelines/1299406095904116742"}
```

Comment:
This seems much messier than it should be. In particular, I have two concerns:

1. It seems incorrect to me that we're restricting the English-language twitter feed to only the English _language_, but then restricting the Spanish-language twitter feed by _country_. Seems to me like we should just show the English-language twitter feed to everyone, unless they're viewing the page in Spanish in which case we should show the Spanish-language twitter feed.

2. Do we have a specific reason for restricting the "Highlights from previous years" section to English-only? Again, it seems to me that we should just show that everywhere.


I'm imagining something like:

```haml
  .row
    .col-md-8
      %br/
      %h2
        Hour of Code Highlights
        / Show the HoC Spanish Twitter feed for users viewing the page in Spanish, and the English Twitter feed for everyone else
        - timeline_id = ['es', 'la'].include? @language ? "1299406095904116742" : "925449654308704256"
      %a.twitter-grid{data: {tweet: {limit: "1"}, chrome: "noheader nofooter"}, href: "https://twitter.com/codeorg/timelines/#{timeline_id}"}
        Hour of Code Highlights
      %script{async: "", charset: "utf-8", src: "//platform.twitter.com/widgets.js"}
    .col-md-3.highlights-column
      %h2 Highlights from previous years
      %div.highlight-tiles
        - highlights.each do |item|
          = view :highlight_item, item: item
        .clear
        %a{href: "/highlights"}
          %button See more highlights
```

Also, we should translate the various English strings ("Hour of Code Highlights", etc)

Reply:
I'll check w/ Caroline to see if she's okay with showing the Spanish-language twitter feed to Spanish-speakers (instead of LATAM-located users). I think the English twitter feed we should leave as is for now as the tweets themselves are not translatable, though I can check with Marketing to see if that is their intention. Similarly, the previous highlights go to English-language news articles.

Reply:
Looks like they're planning on having Spanish- and Portuguese-language tweets in the tweet deck, with all of the tweets pertaining to LATAM hour of code, which is the reason they wanted it to be LATAM-country specific

Reply:
I see; so this should more accurately be called our "LATAM Hoc Twitter Feed", not our "Spanish Twitter Feed"?

Reply:
I'll change the Twitter feed name as @Hamms suggested. I'll also remove `@country == 'la'` from the LATAM condition.

@dju90 will check with international team to see if they want to continue using `hourofcode.com/la` for LATAM countries in their communication. If we want to use a different 2-letter code for LATAM, we can choose an unused code  from this table https://en.wikipedia.org/wiki/ISO_3166-1_alpha-2#Current_codes.